### PR TITLE
Enable SSL for Postgres Fast sync tap and target connections

### PR DESCRIPTION
**Pull request by alastairstuart**
## Problem

Enables SSL for postgres fastsync tap and targets

## Proposed changes

* Added sslmode='require' to the connection string is ssl = true in the config.

## Types of changes

What types of changes does your code introduce to PipelineWise?

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

- [x] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [x] Description above provides context of the change
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Unit tests for changes (not needed for documentation changes)
- [ ] CI checks pass with my changes
- [ ] Bumping version in `setup.py` is an individual PR and not mixed with feature or bugfix PRs
- [ ] Commit message/PR title starts with `[AP-NNNN]` (if applicable. AP-NNNN = JIRA ID)
- [ ] Branch name starts with `AP-NNN` (if applicable. AP-NNN = JIRA ID)
- [ ] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [ ] Relevant documentation is updated including usage instructions

### CODE REVIEW

**Comment by koszti** [comment]
Related PR for ppw-tap-postgres: https://github.com/transferwise/pipelinewise-tap-postgres/pull/64. We'll need to release a new version to PyPI and bump the postgres tap [here](https://github.com/transferwise/pipelinewise/blob/master/singer-connectors/tap-postgres/requirements.txt).

File: pipelinewise/fastsync/commons/tap_postgres.py [hunk 1]
```diff
@@ -57,6 +57,10 @@ def open_connection(self):
             self.connection_config.get('replica_user', self.connection_config['user']),
             self.connection_config.get('replica_password', self.connection_config['password']),
             self.connection_config['dbname'])
+        
+        if 'ssl' in self.connection_config and self.connection_config['ssl'] == 'true':
```
Comment (koszti) [inline pipelinewise/fastsync/commons/tap_postgres.py:61]: Can we accept not only `true` (string) but logical `True` as well?

**Comment by koszti** [comment]
@alastairstuart can you please bump pipelinewise-tap-postgres to 1.7.0 [here](https://github.com/transferwise/pipelinewise/blob/master/singer-connectors/tap-postgres/requirements.txt) , which is including your recent changes in the tap?

**Comment by alastairstuart** [comment]
Tap Version has been bumped

**Comment by koszti** [comment]
@alastairstuart pipelinewise-target-postgres 2.1.0 has been release to PyPI with your changes.
Can you please bump the target-postgres version to 2.1.0 [here](https://github.com/transferwise/pipelinewise/blob/master/singer-connectors/target-postgres/requirements.txt) ?

Can you please resolve conflicts as well? Seems like there is a tiny issue with your previous commit

**Review by koszti — APPROVED** [review]

**Pull request merged**